scripts/us_census/acs5yr/subject_tables/common/dc_utils.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Commons REST API utils."""
+
+import copy
+import ast
+import json
+import os
+from absl import app
+from absl import flags
+import requests
+import time
+
+module_dir_ = os.path.dirname(__file__)
+_FLAGS = flags.FLAGS
+
+flags.DEFINE_string('dcid', None, 'dcid of the node to query')
+flags.DEFINE_string('dc_output_path', './prefetched_outputs/',
+                    'Path to store the output')
+flags.DEFINE_boolean('force_fetch', False,
+                     'forces api query and not return cached result')
+
+
+def request_url_json(url: str) -> dict:
```

Comment:
Can this be used instead?
https://github.com/datacommonsorg/data/blob/d4d2f8600ce947c436e8ad630e07d769f2a0cf87/tools/download_utils/requests_wrappers.py#L24

Reply:
Done. Thanks for pointing this out.